### PR TITLE
Changes To leveler, Warnings, and chatchart

### PR DESCRIPTION
**Pull request by DannyDB5544**
Leveler has had some Typos Corrected, Warnings has more functional warntimes, and Chatchart is a little more verbose (titles in the plotchart now detail the amount of messages)

### CODE REVIEW

File: badge/badge.py [hunk 1]
```diff
@@ -38,26 +38,31 @@ async def issuesteambadge(self, ctx, steam_id: str):
...
-                        "badge": "10",
-                        "info": ctx.message.author.name,
-                        "info2": ctx.message.author.id,
-                    },
-                )
-                await ctx.send(issue_request.text)
-            else:
-                await ctx.send("You currently have an active badge.")
+        try:
+            if ctx.message.guild.id == 330432627649544202: #Replaced with NW Guild check
```
Comment (AgentBlackout) [inline badge/badge.py:42]: Can be moved to a short circuit check to reduce nesting

File: badge/badge.py [hunk 2]
```diff
@@ -70,25 +75,30 @@ async def issuediscordbadge(self, ctx):
...
-                        "token": await self.config.TOKEN(),
-                        "action": "issue",
-                        "id": (str_id + "@discord"),
-                        "badge": "10",
-                        "info": ctx.message.author.name,
-                        "info2": str_id,
-                    },
-                )
+        try:
+            if ctx.message.guild.id == 330432627649544202: #Replaced with NW Guild check
```
Comment (AgentBlackout) [inline badge/badge.py:79]: short circuit check here

File: badge/badge.py [hunk 3]
```diff
@@ -70,25 +75,30 @@ async def issuediscordbadge(self, ctx):
...
+                            "id": (str_id + "@discord"),
+                            "badge": "10",
+                            "info": ctx.message.author.name,
+                            "info2": str_id,
+                        },
+                    )
+
+                    await ctx.send(issue_request.text)
+                else:
+                    await ctx.send("You currently have an active badge")
```
Comment (AgentBlackout) [inline badge/badge.py:97]: Logic for both of these can probably be consolidated into a single method

**Comment by DannyDB5544** [comment]
TODO: - Ensure badges cog works, ensure baseline functionality pending short move of 079 to an updated instance (in progress)

**Comment by DannyDB5544** [comment]
Right, Threw in a quick tweak for Leveler too - new command, mostly, !bal but for reputation - !reps

File: leveler/leveler.py [hunk 4]
```diff
@@ -959,7 +986,7 @@ async def info(self, ctx, *, info):
             await db.users.update_one(
                 {"user_id": str(user.id)}, {"$set": {"info": info}}
             )
-            await ctx.send("**Your info section has been succesfully set!**")
+            await ctx.send("**Your info section has been successfully set!**")
```
[the remark this answers is not in the document]
Comment (DannyDB5544) [reply leveler/leveler.py:989]: Murdered successfully

**Pull request merged**